front50-core/src/main/java/com/netflix/spinnaker/front50/model/plugins/PluginInfo.java

```diff
@@ -114,6 +121,22 @@ public void setReleaseByVersion(String version, Release release) {
      */
     private String requires;
 
+    @JsonIgnore
```

Comment:
is this reasonable?  This could be a static method instead.

Reply:
I think I prefer it not static.